### DISCUSSION TITLE
No GPU support on official docker image?

### DISCUSSION THREAD

**Discussion by gald89** [pycaret/pycaret#2290] [General]
I am currently using the full pycaret docker [image](https://hub.docker.com/r/pycaret/full). I have noticed that Lightroom GPU support seems to be lacking since I was getting a number of errors such as 'LightGBMError : GPU Tree Learner was not enabled in this build' and performance load on GPU is quite low. Is that a way hope to enable this on the image?

**Comment by rakovskij-stanislav** [comment]
@gald89 Probably it would be better to use [`cschranz/gpu-jupyter` ](https://github.com/iot-salzburg/gpu-jupyter) (default command):
```
docker run --gpus all -d -it -p 8848:8888 -v $(pwd)/data:/home/jovyan/work -e GRANT_SUDO=yes -e JUPYTER_ENABLE_LAB=yes --user root cschranz/gpu-jupyter:v1.6_cuda-12.0_ubuntu-22.04
```

Move to console (available in jupyter) and just install required packages:

```bash
sudo apt update && sudo apt install -y python3 python3-pip libgomp1
pip install --upgrade setuptools pip
pip install pycaret[analysis,models,mlops] --ignore-installed PyYAML==6.0.1
```

lightgbm will work in CPU mode (too much things to run it properly in GPU mode), but other models works nice
